axis_backend/axis_backend/settings/production.py

```
@@ -4,10 +4,135 @@
+            'handlers': ['console'],
+            'level': 'INFO',
+            'propagate': False,
+        },
+    },
+}
+
+# Cache - Use Redis in production
+CACHES = {
+    'default': {
+        'BACKEND': 'django.core.cache.backends.redis.RedisCache',
+        'LOCATION': REDIS_URL,
```

Ruff (0.14.5)

`REDIS_URL` may be undefined, or defined from star imports

(F405)